src/mainboard/raptor-cs/talos-2/Kconfig

```
@@ -27,6 +27,23 @@ config BOARD_SPECIFIC_OPTIONS
 	select IPMI_BT_ROMSTAGE
 	select FLATTENED_DEVICE_TREE
 
+config TALOS_2_INFINEON_TPM_1
+	bool "I2C SLB9635TT or SLB9645TT or another TPM1 chip compatible with SLB9635TT"
```

This doesn't fit in menuconfig launched in standard 80x25 shell window
```suggestion
	bool "I2C TPM1 chip compatible with SLB9635TT"
```